library/time/deconstruct_keys_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../../spec_helper'
+require 'time'
```

Comment:
**major:** The method `Time#deconstruct_keys` is available without requiring `time`, so it's a Core method. So this file should be placed in `core/time/` directory.